src/explore-education-statistics-common/src/styles/_tables.scss

```diff
@@ -81,3 +81,9 @@ table,
     border-left-color: govuk-colour('red');
   }
 }
+
+.dfe-table-vertical-align-middle {
```

Comment:
Naming could be improved to follow BEM conventions i.e. `.dfe-table--vertical-align-middle`